api/client/service/ps.go

```diff
@@ -14,20 +14,20 @@ import (
 )
 
 type psOptions struct {
-	serviceID string
-	noResolve bool
-	filter    opts.FilterOpt
+	serviceIDs []string
```

Comment:
These aren't just service ids. These can also be the names of services. Make sure to model that correctly.